pgbouncer/assets/configuration/spec.yaml

```diff
@@ -8,7 +8,11 @@ files:
   - template: instances
     options:
       - name: database_url
-        description: "`database_url` parameter should point to PgBouncer stats database url"
+        description: |
+          The PgBouncer stats database URL.
+
+          If the password contains special characters, be sure to escape them using
+          URL encoding (also known as percent-encoding).
```

Comment:
Would it be interesting to add an example or a link here?

Reply:
Would linking to this sound OK?

https://developer.mozilla.org/en-US/docs/Glossary/percent-encoding

Reply:
This would also be a good link: https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING